go.mod

```diff
@@ -133,7 +133,7 @@ require (
 	github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
 	github.com/urfave/cli/v2 v2.8.1
 	github.com/whyrusleeping/bencher v0.0.0-20190829221104-bb6607aa8bba
-	github.com/whyrusleeping/cbor-gen v0.0.0-20220514204315-f29c37e9c44c
+	github.com/whyrusleeping/cbor-gen v0.0.0-20221021053955-c138aae13722
```

Comment:
We now have a proper release for this, will push a commit shortly.

Reply:
Still TODO

Reply:
I believe this was supposed to be pointing to go-jsonrpc and not to cbor-gen. That repo does not do releases.